src/rust/engine/serverset/src/lib.rs

```diff
@@ -404,9 +404,14 @@ mod tests {
 
     runtime.block_on(s.next()).unwrap();
 
-    // The serverset is configured to block for 10ms; make sure we waited for at least 9ms for
-    // stability.
-    assert!(start.elapsed() > Duration::from_millis(9))
+    // The serverset is configured to block for 20ms; make sure we waited for at least 10ms for
```

Comment:
I don't see the change that makes this go from 10 to 20 ms. Can you explain what you have in mind?

Reply:
Added comment